src/Lime.Protocol.UnitTests/Serialization/Newtonsoft/EnvelopeSerializerTests.cs

```diff
@@ -1450,6 +1501,55 @@ public void Deserialize_TextMessage_ReturnsValidInstance()
             Assert.AreEqual(text, textContent.Text);
         }
 
+        [Test]
+        [Category("Deserialize")]
+        public void Deserialize_ReplyMessage_ReturnsValidInstance()
+        {
+            var target = GetTarget();
+
+            var id = EnvelopeId.NewId();
+            var from = Dummy.CreateNode();
+            var pp = Dummy.CreateNode();
+            var to = Dummy.CreateNode();
+
+            string randomKey1 = "randomString1";
+            string randomKey2 = "randomString2";
+            string randomString1 = Dummy.CreateRandomStringExtended(50);
+            string randomString2 = Dummy.CreateRandomStringExtended(50);
+
+            var text = Dummy.CreateRandomStringExtended(50);
+            var text1 = text.Escape();
+            var text2 = text.Escape();
```

Comment:
You should create another dummy text for the replyTo content, since they will most probably be different. 

The same idea was used for Metadata random strings in this (and other) test, btw.